File: AppDemo/AppDemo/ViewController.m
//
//  ViewController.m
//  AppDemo
//
//  Created by ZP on 2021/5/13.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
}

int sum(int a,int b) {
    return a + b;
}

- (void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    NSLog(@"sum result: %d",sum(10, 20));
}

@end

File: SwiftAppDemo/SwiftAppDemo/ViewController.swift
//
//  ViewController.swift
//  SwiftAppDemo
//
//  Created by ZP on 2021/5/13.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
    }

    func sum(a: Int, b: Int) -> Int {
        return a + b
    }

    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
        print(sum(a: 10, b: 20))
    }
}
